process_preds.Rmd

```diff
@@ -32,18 +32,16 @@ output_path <- "data/input_processed"
 ## set the 'template' crs and extent and write a function that matches these for predictors
 ```{r}
 # write function that reads in predictors and matches CRS and extent to the template
-process_file <- function(input_path, template_path, resolution, save = FALSE, output_path = "data/input_processed") {
-  # Read in the template .shp and save as an sf object
-  template <- read_sf(template_path)
-  assign("template_sf", template, envir = .GlobalEnv)
+process_preds <- function(predictor_path, template, resolution, save, output_path) {
   
   # Initialize variables to hold the processed objects
```

Comment:
We don't necessarily need to initialize these variables, so in the function script you could cut this section out.